main.tex

```diff
@@ -105,28 +105,28 @@ \subsubsection{Block-space efficiency}
 
 \subsection{Our Contribution}
 
-We introduce WabiSabi, a generalization of Chaumian CoinJoin based on a Keyed-Verification Anonymous Credentials-based (KVAC) scheme~\cite{chase2019signal}. The use of KVACs replaces blind signatures' standard denominations with homomorphic amount commitments, similar to Confidential Transactions~\cite{maxwell2016confidential}, but retains the invariant that the sum of any one participant's outputs does not exceed that of their inputs without the coordinator learning the underlying values. In addition to being more flexible this improves privacy compared to standard denominations, since smaller inputs can be combined and change outputs created with the same unlinkability guarantees as the privacy enhanced outputs\footnote{Note that the cleartext amounts appearing in the final transaction might still link individual inputs and outputs.}.
+We present WabiSabi, a generalization of Chaumian CoinJoin based on a Keyed-Verification Anonymous Credentials-based (KVAC) scheme~\cite{chase2019signal}. The use of KVACs replaces blind signatures' standard denominations with homomorphic amount commitments, similar to Confidential Transactions~\cite{maxwell2016confidential} in that that the sum of any one participant's outputs does not exceed that of their inputs while hiding the underlying values from the coordinator. In addition to being more flexible this improves privacy compared to standard denominations, since smaller inputs can be combined and change outputs created with the same unlinkability guarantees as the privacy enhanced outputs\footnote{Note that the cleartext amounts appearing in the final transaction might still link individual inputs and outputs.}.
 
-WabiSabi can be instantiated to construct a variety of CoinJoin transaction structures that depart from the standard output denomination convention, such as SharedCoin\footnote{\url{https://github.com/sharedcoin/Sharedcoin}} and CashFusion\footnote{\url{https://github.com/cashshuffle/spec}} style transactions and Knapsack~\cite{maurer2017anonymous} mixing. Additionally it makes possible arbitrary consolidation of outputs, minimizing unmixed change, relaxing minimum required denominations, improved block space efficiency, making payments from CoinJoins, as well as payments within CoinJoins, also known as PayJoins.
+WabiSabi can be instantiated to construct a variety of CoinJoin transaction structures that depart from the standard output denomination convention, used by SharedCoin\footnote{\url{https://github.com/sharedcoin/Sharedcoin}} and CashFusion\footnote{\url{https://github.com/cashshuffle/spec}} style transactions and Knapsack~\cite{maurer2017anonymous} mixing. Additionally it makes arbitrary consolidation of outputs possible, minimizes unmixed change, relaxes minimum required denominations, improves block space efficiency, and makes payments from as well as within CoinJoins, also known as PayJoins, possible.
```

Comment:
i think this is a bit less clear than the previous version, and this is exacerbated by my loose use of "payjoin" in the previous version.

i think a precise use is:

coinjoin = multiparty txn with the anonymity property
equal amount coinjoin = some subset of outputs having the same amounts
payjoin = steganographic coinjoin between 2 parties using arbitrary amounts and a net value transfer between parties

so what this makes possible in general is coinjoins, with few restrictions. we can require or allow equal amount coinjoins. if arbitrary amounts are allowed than arbitrary payments like joinmarket's send scripts can be done. better yet, value can be exchanged through credentials, allowing privacy enhanced net value transfers within a coinjoin, that have similarities to payjoins but which are not steganographic. the reason this is notable is that the ~0 balance assumption of the non-derived sub-transaction becomes much stronger with that as a possibility.

Reply:
you're right. I misread this section when editing and the changing such as to using gets especially unclear

Reply:
```suggestion
WabiSabi can be instantiated to construct a variety of CoinJoin transaction structures that depart from the standard output denomination convention, used by SharedCoin\footnote{\url{https://github.com/sharedcoin/Sharedcoin}} and CashFusion\footnote{\url{https://github.com/cashshuffle/spec}} style transactions and Knapsack~\cite{maurer2017anonymous} mixing. Payments from CoinJoin transactions are possible, as are payments within them making this potentially a multiparty PayJoin that trades the steganographic properties for improved privacy from counterparties. Additionally, restrictions on consolidation of inputs can be removed, and there are opportunities for reducing unmixed change and relaxing minimum required denominations, and improved block space efficiency.
```